src/asmjs/CMakeLists.txt

```diff
@@ -2,4 +2,7 @@ SET(asmjs_SOURCES
   asm_v_wasm.cpp
   shared-constants.cpp
 )
-ADD_LIBRARY(asmjs STATIC ${asmjs_SOURCES})
\ No newline at end of file
+ADD_LIBRARY(asmjs STATIC ${asmjs_SOURCES})
```

Comment:
We need to compile everything that gets linked into the shared library with -fPIC, on all platforms. It's only just lucky that things work now (because on x86-64 compilers prefer rip-relative addressing even when not generating strictly PIC code).